src/styles/blocks/explore-online.scss

```diff
@@ -0,0 +1,69 @@
+.explore-online {
+  @include onTablet {
+    max-width: 536px;
+  }
+
+  @include onDesktop {
+    max-width: none;
+  }
+
+  &__title {
+    margin-bottom: 40px;
+  }
+
+  &__performance-wrapper {
+    @include onDesktop {
+      display: flex;
+    }
+  }
+
+  &__performance {
+    margin-bottom: 48px;
+    &:last-child {
+      margin-bottom: 76px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 89px;
+    }
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto 24px;
+    width: 280px;
+
+    transition: transform 0.5s;
+
+    &:hover {
+      transform: scale(1.05);
+    }
+
+    @include onTablet {
+      width: 392px;
+    }
+
+    @include onDesktop {
+      width: 324px;
+      margin: 0 24px 32px 0;
+      
+      .explore-online__performance:last-child & {
```

Comment:
Use :last-of-type instead of :last-child to target the last element with the same class in the container.